live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/ports/storage/StorageClient.java

```diff
@@ -23,17 +23,14 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  * #L%
  */
-package org.alfresco.hxi_connector.live_ingester.storage;
+package org.alfresco.hxi_connector.live_ingester.domain.ports.storage;
 
 import java.io.File;
 import java.io.InputStream;
-import java.net.URL;
-
-import org.apache.http.StatusLine;
 
 public interface StorageClient
 {
-    StatusLine upload(File file, String contentType, URL preSignedUrl);
+    void upload(File file, String contentType, String nodeId);
```

Comment:
are you sure we need this method?

Reply:
I'm not sure if data will be passed as `File` or `InputStream`, so I've prepared form both possibilities